modules/openapi-generator/src/main/resources/elixir/model.mustache

```diff
@@ -4,34 +4,15 @@
   {{&description}}
   """
 
-  @derive [Poison.Encoder]
+  @derive Jason.Encoder
   defstruct [
-    {{#vars}}{{#atom}}{{&baseName}}{{/atom}}{{^-last}},
+    {{#vars}}{{#atom}}{{name}}{{/atom}}{{^-last}},
```

Comment:
This changes the name and writing of the payload, which results in breaking the spec due to diverging of the property name.

see generated examples.